src/gui/folder.cpp

```diff
@@ -163,8 +163,8 @@ Folder::Folder(const FolderDefinition &definition, const AccountStatePtr &accoun
 
         // Potentially upgrade suffix vfs to windows vfs
         OC_ENFORCE(_vfs);
-        // Initialize the vfs plugin
-        startVfs();
+        // Initialize the vfs plugin. Do this after the UI is running, so we can show a dialog when something goes wrong.
+        QTimer::singleShot(0, this, [this] { startVfs(); });
```

Comment:
I don't think we need a lambda here, a direct connect should be possible.

Reply:
`startVfs()` is not a slot...

Reply:
🤦